_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+id: Custom Resource Definition
+name: Custom Resource Definition
+tags:
+- fundamental
+- operation
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
```

Comment:
Would also add `aka` below this. See https://github.com/kubernetes/website/blob/master/_data/glossary/_example.yml#L4.

Points under  `aka`:

- CRD
- Formerly known as Third Party Resource (TPR)

Reply:
Also, `related` under aka. See https://github.com/kubernetes/website/blob/master/_data/glossary/_example.yml#L8.

Points under `related`:

- id for the glossary of custom resources (please create a dummy link here, we can add the glossary for custom resources later)

Reply:
Hi @nikhita ! Thanks for providing all these great suggestions. [The _example.yml](https://github.com/kubernetes/website/blob/master/_data/glossary/_example.yml#L8) was particularly helpful. 😄

Reply:
I'll leave the related section aside for now until someone responds to your request on this [issue](https://github.com/kubernetes/website/issues/5993). However I would agree that custom resources should have its own glossary definition 😄

Reply:
> I'll leave the related section aside for now until someone responds to your request on this issue. 

Sure, makes sense. 👍